<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Random Dice</string>
  </property>
  <widget class="QGraphicsView" name="land_09">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Dice_1</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="player2_text">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>650</y>
     <width>350</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_10">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_13">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_11">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_12">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_08">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_07">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_06">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_05">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_03">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_04">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_02">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>460</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_01">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>460</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_00">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>460</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_14">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_15">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_17">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="land_16">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>260</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="player1_text">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>650</y>
     <width>350</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../_image/player1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>260</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../_image/player2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../_image/dice.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="player1">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../_image/player1_resize.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="player2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>510</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../_image/player2_resize.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Dice_2</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="main_text">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>200</width>
     <height>90</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections/>
</ui>
